app/components/dialog/components/modifyProfile/index.tsx

```diff
@@ -94,88 +93,102 @@ class ModifyProfileDialog extends React.PureComponent<
-                <label htmlFor="currentAffiliation">Current Affiliation</label>
-                <Field
-                  name="currentAffiliation"
-                  component={AffiliationSelectBox}
-                  inputClassName={styles.inputField}
-                  format={this.formatAffiliation}
-                />
-              </div>
-            </div>
-            <div className={styles.bioWrapper}>
-              <label htmlFor="bio">
-                Short Bio<small> (Optional)</small>
-              </label>
-              <Field
-                name="bio"
-                component={ReduxAutoSizeTextarea}
-                type="text"
-                textareaClassName={styles.textAreaWrapper}
-                textareaStyle={{ padding: "8px" }}
-                placeholder="Please tell us about yourself."
-              />
-            </div>
-            <div className={styles.formControl}>
-              <div className={styles.inlineInput}>
-                <label htmlFor="email">Email Address</label>
-                <Field
-                  inputClassName={styles.inputField}
-                  name="email"
-                  component={ScinapseReduxInput}
-                  type="text"
-                  placeholder="Email Address"
-                />
-              </div>
-              <div className={styles.inlineInput}>
-                <label htmlFor="website">
-                  Website URL<small> (Optional)</small>
-                </label>
-                <Field
-                  inputClassName={styles.inputField}
-                  name="website"
-                  component={ScinapseReduxInput}
-                  type="text"
-                  placeholder="e.g. https://username.com"
-                />
-              </div>
-            </div>
-          </div>
-          <div className={styles.footer}>
-            <div className={styles.buttonsWrapper}>
-              <ScinapseButton
-                type="submit"
-                style={{
-                  backgroundColor: isLoading ? "#ecf1fa" : "#6096ff",
-                  cursor: isLoading ? "not-allowed" : "pointer",
-                  width: "127px",
-                  height: "40px",
-                  fontWeight: 500,
-                  fontSize: "16px",
-                }}
-                disabled={isLoading}
-                gaCategory="New Author Show"
-                gaAction="Click Save Button in Edit Profile "
-                gaLabel={`Save author : ${author.id} profile`}
-                content="Save Changes"
-              />
-            </div>
-          </div>
-        </form>
+        <Formik
+          ref={(el: any) => (this.formikNode = el)}
+          initialValues={initialValues}
+          onSubmit={handleSubmitForm}
+          validate={validateForm}
+          enableReinitialize={true}
+          render={({ handleSubmit }) => {
+            return (
+              <form onSubmit={handleSubmit}>
```

Comment:
You can use \<Form\> from "Formik". It helps you reduce code.

https://jaredpalmer.com/formik/docs/api/form